MapboxCoreNavigation/RouteController.swift

```diff
@@ -692,6 +688,12 @@ extension RouteController: CLLocationManagerDelegate {
             if secondsToEndOfStep <= RouteControllerHighAlertInterval {
```

Comment:
Can you provide a table similar to the ones in #427, so we can get a sense of the speeds that trigger the distance thresholds versus the speeds that trigger the time thresholds?